cli/lib/context/context_git_get_remote_url.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:stax/context/context.dart';
+
+extension ContextGitGetRemoteUrl on Context {
+  String? getRemoteUrl() {
+    final remote = git.remote.runSync().stdout.toString().trim();
```

Comment:
On main branch there is one bug fix for a case when user has more then one remote. 
```suggestion
    final remote = context.git.remote.runSync().stdout.toString().split('\n')[0].trim();
```